src/Microsoft.Win32.SystemEvents/src/PinvokeAnalyzerExceptionList.analyzerdata

```diff
@@ -0,0 +1,25 @@
+wtsapi32.dll!WTSRegisterSessionNotification
```

Comment:
Name file PinvokeAnalyzerExceptionList.analyzerdata.netcoreapp so it only applies to that TFM

Reply:
It's being used for the netstandard-Windows_NT configuration.  I can name it specific to that.

Reply:
Per @weshaggard I will retarget to netcoreapp.